client/dist/index.html

```diff
@@ -7,6 +7,7 @@
 </head>
 <body>
 <div id="app"></div>
+<div id="overview"></div>
```

Comment:
Why did you not have your view render in app? Given this change, we would each have to add a new div for our components instead of rendering them in div.

Is this just for ease of development, but you then intend to refactor?